docroot/themes/humsci/humsci_basic/src/scss/components/_card.scss

```diff
@@ -164,6 +168,8 @@
   }
 
   &__decorative-link {
+    padding: 1rem 1.275rem 0.5rem 0; // 44px WCAG AAA target size
```

Comment:
Instead of adjusting the padding here, let's set a height and width on the link to be 44x44 and make the SVG a themed background images. We didn't have the SVG function when this was originally styled and now would be a good time to refactor.